properties.schema

```diff
@@ -3,6 +3,16 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "id": "http://jsonschema.net",
   "required": false,
+  "globals": {
+    "incompleteContent": {
+      "type": "string",
+      "required": true,
+      "default": "",
```

Comment:
Missing default text? Looks like we usually fill this in e.g. [here](https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/blob/master/properties.schema#L42).